pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -532,7 +552,32 @@ func (p *MasqueradePodInterface) discoverPodNetworkInterface() error {
 		return fmt.Errorf("failed to parse vm ip address %s", vm)
 	}
 	p.vif.IP = *vmAddr
+	return nil
+}
 
+func configureVifV6Addresses(p *MasqueradePodInterface, err error) error {
+	if p.vmIpv6NetworkCIDR == "" {
+		p.vmIpv6NetworkCIDR = api.DefaultVMIpv6CIDR
+	}
+
+	defaultGatewayIpv6, vmIpv6, err := Handler.GetHostAndGwAddressesFromCIDR(p.vmIpv6NetworkCIDR)
+	if err != nil {
+		log.Log.Errorf("failed to get gw and vm available ipv6 addresses from CIDR %s", p.vmIpv6NetworkCIDR)
+		return err
+	}
+
+	gatewayIpv6Addr, err := Handler.ParseAddr(defaultGatewayIpv6)
+	if err != nil {
+		return fmt.Errorf("failed to parse gateway ipv6 address %s", gatewayIpv6Addr)
```

Comment:
Print the error too either as "%v" or by using github.com/pkg/errors and errors.Wrapf

Reply:
done